File: CoreLib/src/main/java/org/hexnibble/corelib/commands/rc/NullRC.java
package org.hexnibble.corelib.commands.rc;

/**
 * Null RC. Does nothing.
 */
public class NullRC extends RC {
   public NullRC() {
      super("NullRC");
   }

   @Override
   protected void processCommand() {
      setCommandStatus(COMMAND_STATUS.COMMAND_SUCCESSFULLY_COMPLETED);
   }
}

File: CoreLib/src/main/java/org/hexnibble/corelib/commands/rc/NullRC.kt
package org.hexnibble.corelib.commands.rc

/**
 * Null RC. Does nothing.
 */
class NullRC(commandID: String = "NullRC") : RC(commandID) {
  override fun processCommand() {
    commandStatus = COMMAND_STATUSES.COMMAND_SUCCESSFULLY_COMPLETED
  }
}